.github/workflows/build.yml

```diff
@@ -1,34 +1,33 @@
 name: Lint/Test/Build
+env:
+  NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need the token on this job? The package is public and nothing here appears to publish/require it

Reply:
No it's not needed. I just have got in the habit of adding it. I'll remove.

Reply:
removed